lib/srv/db/audit_test.go

```diff
@@ -240,6 +241,12 @@ func waitForEvent(t *testing.T, testCtx *testContext, code string) events.AuditE
 		select {
 		case event := <-testCtx.emitter.C():
 			if event.GetCode() != code {
+				// ignored events may be helpful in debugging test failures
+				bytes, err := json.Marshal(event)
+				if err != nil {
+					bytes = []byte(err.Error())
+				}
```

Comment:
Why we are handling go error like a valid event content  - `json=%v` ?

Reply:
This only goes to the log, and we want a string representation of the event; if we cannot get it for any reason (very unlikely!), then a string representation of error is a valuable thing too.